### PR TITLE
fix TS errors

### PR DESCRIPTION
**Pull request by konstantin-kitmanov**
First of all, thank you for the great component which saved quite a lot of time.

But while building a Nuxt 3 project I noticed a few typescript errors, despite the component working as expected as far as I can tell. Hopefully this helps.

### CODE REVIEW

File: src/composable.ts [hunk 1]
```diff
@@ -420,49 +423,49 @@ const useVuePopper = function (suppliedOptions: Options = {}) {
 										'data-placement': placementState.value,
 										'data-name': getTransitionNames.value.inner,
 
-										onBeforeEnter(element: HTMLElement) {
```
Comment (abdul-alhasany) [inline src/composable.ts:423]: What benefit does this provide over the current implementation?

**Comment by abdul-alhasany** [comment]
Thanks for the contribution. Do you have a sample project that I can test these changes against?

**Comment by konstantin-kitmanov** [comment]
Unfortunately, I can't provide a sample project, since the one I'm working on currently is a closed-source one.

**Comment by abdul-alhasany** [comment]
No problem. Since tests are passing, can you update the version number so I can merge

**Comment by konstantin-kitmanov** [comment]
Added commit with patch version updated.

**Pull request merged**